src/__tests__/element-queries.js

```diff
@@ -422,21 +427,27 @@ test('queryAllByRole returns semantic html elements', () => {
   expect(queryAllByRole(/columnheader/i)).toHaveLength(1)
   expect(queryAllByRole(/rowheader/i)).toHaveLength(1)
   expect(queryAllByRole(/grid/i)).toHaveLength(1)
-  expect(queryAllByRole(/form/i)).toHaveLength(1)
+  expect(queryAllByRole(/form/i)).toHaveLength(0)
   expect(queryAllByRole(/button/i)).toHaveLength(1)
   expect(queryAllByRole(/heading/i)).toHaveLength(6)
   expect(queryAllByRole('list')).toHaveLength(2)
   expect(queryAllByRole(/listitem/i)).toHaveLength(3)
-  expect(queryAllByRole(/textbox/i)).toHaveLength(2)
+  // TODO: with https://github.com/A11yance/aria-query/pull/42
+  // the actual value will match `toHaveLength(2)`
```

Comment:
Can be verified by using https://pkg.csb.dev/eps1lon/aria-query/commit/44b2c4e3/aria-query as the `aria-query` version.